cordformation/src/test/resources/gradle.properties

```diff
@@ -7,3 +7,4 @@ jolokia_version=1.6.0
 docker_image_name=corda/corda-zulu-4.3
 
 artifactory_contextUrl=https://software.r3.com/artifactory
+publicArtifactURL=https://download.corda.net/maven
```

Comment:
The `cordformation` plugin should have been retired in the `master` branch :thinking:.